test/cypress/integration/29-expenses.test.js

```diff
@@ -152,6 +153,128 @@ describe('New expense flow', () => {
+      cy.get('input[name="description"]').type('Brussels January team retreat');
+
+      // Upload 2 files to the multi-files dropzone
+      cy.fixture('images/receipt.jpg').then(fileContent => {
+        const getFile = idx => ({ fileContent, fileName: `receipt${idx}.jpg`, mimeType: 'image/jpeg' });
+        const files = [getFile(1), getFile(2)];
+        cy.getByDataCy('expense-multi-attachments-dropzone').upload(files, { subjectType: 'drag-n-drop' });
+      });
+      cy.getByDataCy('expense-attachment-form').should('have.length', 2);
+      cy.getByDataCy('expense-summary-btn').should('be.disabled');
+
+      // Fill info for first attachment
+      cy.get('input[name="items[0].description"]').type('Fancy restaurant');
+      cy.get('input[name="items[0].amount"]').type('{selectall}13');
+      cy.getByDataCy('expense-summary-btn').should('be.disabled');
+      cy.get('input:invalid').should('have.length', 2); // Missing attachment desctiption+amount
+      cy.getByDataCy('expense-items-total-amount').should('contain', '--.--'); // amount for second item is missing
+
+      // Select Payout Method
+      cy.getByDataCy('payout-method-select').click();
+      cy.contains('[data-cy="select-option"]', 'New custom payout method').click();
+      cy.get('[data-cy="payout-other-info"]').type('A simple thanks would work');
+      // cy.get('input[name="privateMessage"]').type('A simple thanks would work');
+      cy.get('input[name="items[1].description"]').type('Potatoes for the giant raclette');
+      cy.get('input[name="items[1].amount"]').type('{selectall}2.50');
+      cy.getByDataCy('expense-summary-btn').click();
+
+      // Submit!
+      cy.getByDataCy('submit-expense-btn').click();
+      cy.contains('[data-cy="temporary-notification"]', 'Expense submited!');
+      cy.contains('[data-cy="expense-page-content"]', 'Brussels January team retreat');
+      cy.getByDataCy('dismiss-temporary-notification-btn').click();
+      cy.getByDataCy('temporary-notification').should('not.exist');
+
+      // Donate some money
+      cy.visit(`/new-collective${collectiveSlug}/donate`);
+      cy.contains('#contributeAs > label', 'A new organization').click();
+
+      // Name must be shown on step
+      cy.get('#contributeAs input[name=name]').type('Evil Corp');
+      cy.get('.step-contributeAs').contains('Evil Corp');
+
+      // Fill form
+      cy.get('#contributeAs input[name=website]').type('https://www.youtube.com/watch?v=oHg5SJYRHA0');
+      cy.get('#contributeAs input[name=githubHandle]').type('test');
+      cy.get('#contributeAs input[name=twitterHandle]').type('test');
+
+      // Submit form
+      cy.contains('button:not([disabled])', 'Next step').click();
+      cy.get('input[type=number][name=custom-amount]').type('{selectall}1337');
+
+      cy.contains('button:not([disabled])', 'Next step').click();
+      cy.wait(5000);
+      cy.fillStripeInput();
+      cy.contains('button', 'Make contribution').click();
+      cy.wait(5000);
```

Comment:
Same feedback about `createHostedCollective` and too much code for the setup. There should not be any test in a `before`, the goal is just to prepare the data - not to test the flow.